docs/events/content/view_content_listing.md

```diff
@@ -6,29 +6,33 @@ title: View Content Listing
 
 Fire whenever a content listing (e.g. Article Card Section, Article List View, etc.) appears on a page.
 
+Pairs with the following event:
+- [Select Content](../../events/content/select_content.md)
+
+
 ## Javascript Code
 
 ```js
 // When:
-// User views an article, blog post, or other content-focused page
+// User views an article, blog post, or other content-focused pages
```

Comment:
suggested: User views **a listing of** articles, article cards, etc.

Reply:
cool, ill update this and others where applicable